src/views/App/index.tsx

```diff
@@ -72,8 +72,12 @@ const AppRoutes = () => {
       <SecureRoute path="/my-requests" component={MyRequests} />
 
       {/* 508 / Accessibility Team Routes */}
-      <Redirect exact from="/508" to="/508/making-a-request" />
-      <SecureRoute path="/508" component={Accessibility} />
+      {flags.hide508Workflow && (
```

Comment:
Should be `!flags.hide508Workflow` here with the routes: "hide" == true -> "don't render the route."

Reply:
good catch, thanks Adam!